projects/picker/src/lib/date-time/timer-box.component.html

```diff
@@ -41,6 +41,7 @@
     (input)="handleInputChange(valueInput.value)"
     (focusin)="focusIn()"
     (focusout)="focusOut(valueInput.value)"
+    onFocus="this.select()"
```

Comment:
Можно обойтись только ангуляром и на focusin сделать valueInput.select()
```suggestion
    (focusin)="valueInput.select(); focusIn();"
```